lib/foo.js

```diff
@@ -2,7 +2,9 @@ var service = require('./service');
 
 exports.bar = function (context, event) {
   if (!event.size) {
+    console.log('Missing parameter: size');
     context.fail('Missing parameter: size');
+    return;
```

Comment:
Nice catch! But if context.fail stops iteration, we have a bug in our tests (not in the handler), maybe the test function faked (fail: function(msg) {}) should raise an exception instead of doing nothing. Do you how it works?